File: dotnet/Core/Workspace/CSharp/tests/context/MultipleSessionContext.cs
namespace Tests.Workspace
{
    public class MultipleSessionContext : Context
    {
        public MultipleSessionContext(Test test) : base(test)
        {
            this.Session1 = test.Workspace.CreateSession();
            this.Session2 = test.Workspace.CreateSession();
        }
    }
}

File: typescript/modules/libs/workspace/adapters/json/system-tests/src/lib/runtime/context/multiple-session-context.ts
import { Fixture } from '../../fixture';
import { Context } from './context';

export class MultipleSessionContext extends Context {
  constructor(fixture: Fixture, name: string) {
    super(fixture, name);
    this.session1 = this.fixture.workspace.createSession();
    this.session2 = this.fixture.workspace.createSession();
  }
}
